libmachine/crashreport/crash_report_test.go

```diff
@@ -0,0 +1,20 @@
+package crashreport
+
+import (
+	"testing"
+
+	"github.com/bugsnag/bugsnag-go"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFileIsNotReadWhenNotExisting(t *testing.T) {
+	metaData := bugsnag.MetaData{}
+	addFile("not existing", &metaData)
+	assert.Equal(t, 0, len(metaData))
+}
+
+func TestRead(t *testing.T) {
+	metaData := bugsnag.MetaData{}
+	addFile("../../test/_resources/vbox.log.txt", &metaData) // me no like this path :(
```

Comment:
Instead of this hardcoded test file, why not use `ioutil.TempFile`? https://golang.org/pkg/io/ioutil/#TempDir

e.g., something like:

```go
f, err := ioutil.TempFile("", "")
if err != nil {
    t.Fatal(err)
}

logdata := []byte(`
VM started.
VM failling.
`)

if err := ioutil.WriteFile(f.Name(), logdata, 0644); err != nil {
    t.Fatal(err)
}

addFile(f.Name(), &metaData)
assert.Equal(t, logdata, metaData["logfile"][f.Name()])
```

I know it's slightly more code but it strongly decouples the test from having a magic file in the repo.